ui/app/AppLayouts/Chat/views/MembersSelectorView.qml

```diff
@@ -88,23 +88,34 @@ MembersSelectorBase {
             }
 
             if (contactDetails.publicKey === root.rootStore.contactsStore.myPublicKey ||
-                contactDetails.isBlocked) {
+                    contactDetails.isBlocked) {
+                // It's our own pubkey or a blocked user, just ignore
                 root.suggestionsDialog.forceHide = false
                 return
             };
 
             if (contactDetails.isContact) {
+                // Is a contact, we add their name to the list
                 root.pastedChatKey = contactDetails.publicKey
                 root.suggestionsDialog.forceHide = false
                 return
             }
 
-            if (root.model.count === 0 && !root.rootStore.contactsStore.hasPendingContactRequest(contactDetails.publicKey)) {
+            if (root.model.count === 0 &&
+                    !root.rootStore.contactsStore.hasPendingContactRequest(contactDetails.publicKey)) {
+                // List is empty and not a contact yet. Open the contact request popup
                 Global.openContactRequestPopup(contactDetails.publicKey,
                                                popup => popup.closed.connect(root.rejected))
                 return
             }
 
+            if (root.model.count === 0 &&
+                    root.rootStore.contactsStore.hasPendingContactRequest(contactDetails.publicKey)) {
```

Comment:
Minor thing:
`hasPendingContactRequest` internally iterates over all contacts. Here we call it twice, maybe cache the result in a variable?